README.md

```
@@ -24,33 +24,38 @@
 - [x] Landing Page & deployment w/ _Vercel_
 
+
 - [x] Basic Auth w/ _Clerk_
 
+
 - [x] Scaffold Basic UI
 
-  - [x] Hero
-  - [x] Search
-  - [x] Feed
+    - [x] Hero
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)